ubuntudesign/www.ubuntu.com
# remove cloud/snappy for now

## Done

* removed /cloud/snappy post /core release

## QA

* see that /cloud/snappy redirects to /core
* see there is no Snappy in the Cloud menu

## Issue / Card

[trello card](https://trello.com/c/WeKQ4ADw)
